test/Foreign.hs

```diff
@@ -2,7 +2,7 @@
 -- full documentation consult <http://www.netlib.org/blas/>.
 {-# LANGUAGE ForeignFunctionInterface #-}
 module Foreign(
---    isamax,
+    isamax,
```

Comment:
Exported the FORTRAN implementation so that it could be used in benchmarking and unit testing